src/main/java/org/apache/commons/text/FormattableUtils.java

```diff
@@ -46,6 +46,7 @@
      * <p>This constructor is public to permit tools that require a JavaBean
      * instance to operate.</p>
      */
+    @SuppressWarnings("squid:S1118")
```

Comment:
Is that a suppress warning for SonarQube? I am not sure if others are actively looking at the reports produced by SonarQube. I think this is not really necessary, if indeed just for SonarQube.

Reply:
Well, works with Sonaqube and Intellij IDEA.
Therefore I assume Eclipse and Netbeans recognizing it too.

I guess you want it to removed.
True?

Reply:
-1 to this

Eclipse was not displaying any warning before, but with this change it displays this warning: `Unsupported @SuppressWarnings("squid:S1118")`

Reply:
Removed SuppressWarnings anootations completely.